models/yang/annotations/sonic-neighbors-annot.yang

```diff
@@ -0,0 +1,21 @@
+module sonic-neighbors-annot {
+
+    yang-version "1";
+
+    namespace "http://openconfig.net/yang/annotation/snbrs-annot";
+    prefix "snbrs-annot";
+
+    import sonic-neighbors { prefix snbrs; }
+    import sonic-extensions { prefix sonic-ext; }
+
+    deviation /snbrs:sonic-neighbors/snbrs:NEIGH_TABLE {
+        deviate add {
+            sonic-ext:db-name "APPL_DB";
+        }
+    }
+    deviation /snbrs:clear_neighbors {
```

Comment:
clear-neighbor?

Reply:
This call may clear multiple records, therefore it is clear_neighbors.  I can change 'underscore' to a 'hyphen', if that's the naming convention in use. I saw other RPCs and followed their naming convention; for e.g. clear_fdb.